packages/sdk/src/mint/README.md

```diff
@@ -16,42 +16,37 @@ If you want to mint on a v2 smart contract, please [create metadata](../createMe
 
 **As with all new SDK api methods, this call should be wrapped in [execute](../#execute) and passed a signing method. For a guide showing how to make a contract call with mintbase-js click [here](https://docs.mintbase.xyz/dev/getting-started/make-your-first-contract-call-deploycontract)**
 
-## mint(args: MintArgs): NearContractCall
+## mint(args: MintArgsV1): NearContractCall
 
 `mint` takes a single argument of type `MintArgs`
 
-```typescript
-export type MintArgs =  {
-  //the contractId from which you want to mint
-  //can be specified through CONTRACT_ADDRESS configuration / environment variables
-  contractAddress?: string;
-  //the intended owner of the token being minted
-  ownerId: string;
-  // metadata including title, description and reference materials
-  metadata: TokenMetadata;
-  options?: MintOptions;
-  noMedia?: boolean;     // explicit opt-in to NFT without media, breaks wallets
-  noReference?: boolean; // explicit opt-in to NFT without reference
-};
+## MintArgsV1
+
+- **contractAddress (optional):** This is the address of the contract where the token will be minted. If not provided, the TOKEN_CONTRACT environment variable will be used.
+
+- **ownerId:** This is the NEAR account ID of the owner of the token that will be minted.
+
+- **metadata:** This is an object containing the metadata of the token. It includes properties like title, description, media, etc.
 
-export type MintArgs =  {
-  //the contractId from which you want to mint, this can be statically defined via the mbjs config file
+- **royalties (optional):** This is an object that specifies how the funds from the sale of the token will be split among different parties.
+
+- **amount (optional):** This is the number of copies of the token that you want to mint. If not provided, only one copy will be minted.
+
+- **noMedia (optional):** This is a flag that indicates whether the token will be minted without any associated media. If set to true, the token will be minted without any media, which may cause issues with some wallets that expect media to be associated with tokens.
```

Comment:
Not 100% correct:

- If you pass `{reference: {media: null, ...}, ...}` the `mint` method will throw an error.
- `{reference: {media: null, ...}, noMedia: true, ...}` will not throw an error.
- `{reference: {media: "yadda", ...}, noMedia: true, ...}` will create the metadata with `media: "yadda"`.

`noMedia` is basically the "Are you sure" button for minting without media. Same for `noReference`